src/main/scala/tech/cryptonomic/conseil/tezos/NetworkConfigOperations.scala

```diff
@@ -0,0 +1,20 @@
+package tech.cryptonomic.conseil.tezos
+
+import com.typesafe.config.Config
+import tech.cryptonomic.conseil.tezos.PlatformDiscoveryTypes.Network
+import scala.collection.JavaConverters._
+
+object NetworkConfigOperations {
+  /**
+    * Extracts networks from config file
+    *
+    * @param  config configuration object
+    * @return list of networks from configuration
+    */
+  def getNetworks(config: Config): List[Network] = {
```

Comment:
The function fetches a list of platforms, not networks, so it should be called `getPlatforms()`.

Reply:
this function returns networks, the one below, which I've recently added, returns platforms